docs/healthCheck.md

```diff
@@ -0,0 +1,37 @@
+`symbol-bootstrap healthCheck`
+==============================
+
+It checks if the services created with docker compose are up and running.
+    
+- if the docker containers are running.
+- if the services' exposed ports are listening.
+- if the rest gateways' /node/health are OK.
```

Comment:
```suggestion
The performed checks are:
- Whether the docker containers are running.
- Whether the services' exposed ports are listening.
- Whether the rest gateways' /node/health are OK.
```

Reply:
mmm, the "whether" wasn't in the old feedback. 

I cannot commit your change, healthCheck.md is generated. I've changed the typescript source code.

https://github.com/nemtech/symbol-bootstrap/blob/health-check/src/commands/healthCheck.ts#L7

Thanks!